steps/step-signature/lib/angular/template/signature-form.tpl.html

```diff
@@ -0,0 +1,9 @@
+<md-subheader>Provide signature bellow</md-subheader>
```

Comment:
`below`

Reply:
Actually I think it will be better with just `Provide Signature`.